packages/api/src/beacon/routes/beacon/state.ts

```diff
@@ -404,6 +434,28 @@ export function getDefinitions(_config: ChainForkConfig): RouteDefinitions<Endpo
         meta: ExecutionOptimisticAndFinalizedCodec,
       },
     },
+    postStateValidatorIdentities: {
+      url: "/eth/v1/beacon/states/{state_id}/validator_identities",
+      method: "POST",
```

Comment:
I'm curious why this is a `POST` instead of `GET`?

Reply:
same reason as for the getStateValidators api, the `GET` variant is limited due to having to pass pubkeys as part of query which only allows up to 64 per request. See https://github.com/ChainSafe/lodestar/pull/6897 for more context and why we wanna switch to `POST` variant